xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.cpp

```diff
@@ -286,7 +286,7 @@ void CWinRenderer::RenderUpdate(int index, int index2, bool clear, unsigned int
   if (!m_bConfigured)
     return;
 
-  DX::Windowing()->SetAlphaBlendEnable(alpha < 255);
+  DX::Windowing()->SetAlphaBlendEnable(true);
```

Comment:
don't touch this line. 
add
```
DX::Windowing()->SetAlphaBlendEnable(true);
```
after `Render(...)` call

Reply:
thanks!